diagnostic_feedback/tasks.py

```diff
@@ -0,0 +1,130 @@
+"""
+Celery task for CSV student answer export.
+"""
```

Comment:
I have discussed this with Vishal and he believes this might be something that is done independently afterwards.